pages/static/src/scss/helpers/_css-helpers.scss

```diff
@@ -1154,6 +1154,23 @@ a.white-text {
   }
 }
 
+
+.our-dream-bg {
+  background: #356c84 url('../img/our-dream-site-bg.jpg') center;
```

Comment:
Since usually this is the only line that changes for homepage module updates, seems like we should make this more DRY somehow.

Reply:
Correct. This was a "I'll do this one time" thing and turned into an "I'll do this every single time" thing. Should address in future when we reevaluate CSS.